x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/collection_gcp_pub_sub_subscription_creation.json

```diff
@@ -20,8 +20,9 @@
   ],
   "related_integrations": [
     {
+      "integration": "audit",
       "package": "gcp",
-      "version": "1.10.0"
+      "version": "^2.2.1"
```

Comment:
And some were raised

Reply:
I am sure this is from the sorting issue being resolved, but is this variation expected?

Reply:
If we notice this, the reason is because before we were just doing the least compatible in general, instead we are doing the least compatible major version. The ^ is used by Kibana to determine anything with this version and above but only on minor and patch, not major which is why we filter to the latest major.

Reply:
Confirmed this is the expected.
<img width="378" alt="Screen Shot 2022-09-29 at 3 42 26 PM" src="https://user-images.githubusercontent.com/99630311/193127047-c7460c5b-96c1-4a2d-b3c3-7012c0e725fa.png">